services/datalad/datalad_service/tasks/validator.py

```diff
@@ -26,10 +27,10 @@ def validate_dataset_sync(dataset_path, ref):
         process = gevent.subprocess.run(
             ['./node_modules/.bin/bids-validator', '--json', '--ignoreSubjectConsistency', dataset_path], stdout=subprocess.PIPE, timeout=300)
         return json.loads(process.stdout)
-    except subprocess.TimeoutExpired:
+    except subprocess.TimeoutExpired as err:
+        esLogger.log(err)
```

Comment:
This should also handle the case where the error returns quickly. That's anytime the exit code isn't 0. Adding check=True to the subprocess.run arguments will have it throw CalledProcessError that could be handled with the same implementation.